examples/Example1/CMakeLists.txt

```diff
@@ -9,4 +9,4 @@ target_include_directories(example1 PUBLIC
   $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/tracking/inc>
   $<INSTALL_INTERFACE:base>
 )
-target_link_libraries(example1 PRIVATE CUDA::curand VecCore::VecCore CopCore::CopCore)
+target_link_libraries(example1 PRIVATE CUDA::curand VecCore::VecCore VecGeom::vecgeom CopCore::CopCore)
```

Comment:
Is it intentional, or just by mistake?

Reply:
You refer to linking with vecgeom? The geometry is defined there...

Reply:
Yes, but this is Example1 cmake file.

Reply:
This adds the VecGeom include path which is now needed by the (shared) `track.h`. The linking would not be needed, but I didn't try to find a more correct solution.